pgoapi/pgoapi.py

```diff
@@ -534,23 +578,7 @@ def disk_encounter_pokemon(self, lureinfo, retry=False):
             resp = self.disk_encounter(encounter_id=encounter_id, fort_id=fort_id, player_latitude=position[0],
                                        player_longitude=position[1]).call()['responses']['DISK_ENCOUNTER']
             if resp['result'] == 1:
-                capture_status = -1
```

Comment:
Hi @destiny117 I took this change from your PR and pushed it in Develop since i was working on something related, and your PR wasn't merged yet.